### ISSUE TITLE
Add header/footer

### ISSUE THREAD

**Issue by hstandaert** [Jeday/react-docx#10]
I was wondering if I can use the Header and Footer classes and what the syntax would be in react?

From the docx.js documentation: https://docx.js.org/#/usage/headers-and-footers

**Comment by Jeday** [comment]
Yeah I remember thinking how that would work. Ideally you would pass render function with header as a prop to Section.
`<section header={{default:()=>(<header>Hello</header>)}}>` 
But I am not yet sure how strand react nodes work in this render.
 Plus react heavily relies on addChildToParentMethod. In DOM even you pass render function to some component render function will eventually be called,it's result then placed as a child of some node and rendered into DOM. Case with Header and Footer is different because they aren't children to node but rather are complex attributes of it in context of DOCX tree.

**Comment by Jeday** [comment]
But since headers and footers are not that complex you can create them imperatively right in render function(no need for useMemo even since there is only one render run) and pass as prop to section. That is solution right now.

**Comment by Rahul-Lance** [comment]
I'm facing an issue with the import statement of renderAsyncDocument, can you please look into it.
![Screenshot (158)](https://user-images.githubusercontent.com/47268257/120264327-32176900-c2bb-11eb-9e84-cbff89d541ec.png)
